src/frontends/onnx/frontend/src/op/reduce.cpp

```diff
@@ -158,6 +162,30 @@ ov::OutputVector reduce_sum_square(const ov::frontend::onnx::Node& node) {
 }
 
 }  // namespace set_1
+
+namespace set_11 {
+ov::OutputVector reduce_l2(const Node& node) {
+    if (node.get_ov_inputs().at(0).get_partial_shape().rank().is_static()) {
```

Comment:
Not sure it is needed. I suggest to add some missing checks for types. Checks should be aligned with original ONNX documentation.